### ISSUE TITLE
"PostHog Users" Slack Workspace updates

### ISSUE THREAD

**Issue by leggetter** [PostHog/meta#6]
A few suggested updates to improve the experience.

When someone joins send them a DM which asks them to:

- Read the [Code of Conduct](https://posthog.com/docs/contribute/code-of-conduct) (not sure if we can add a checkbox that ensures they've acknowledged as you can in Discord, will need to investigate)
- Introduce themselves in #introductions so we can all get to know a bit more about them. Will provide a template for this too.
- Change the name from "PostHog Users" to "PostHog Community". We refer to this in a few places as the "PostHog Community" and I think it sounds much nicer.

**Comment by piemets** [comment]
Yes! Had a couple of suggestions [here as well](https://github.com/PostHog/company-internal/issues/168) previously.

**Comment by jamesefhawkins** [comment]
We don't have a ton of channels, so I'd like to add that we invite them to the handful we do have so we don't get tempted to use General:

* Editorial, so they can see content without it being annoying in general
* Getting-Started (can't remember if we default to this), since this gets a fair amount of usage
* Bugs (so general doesn't _only_ have bugs reported in it, making the platform feel buggy)

**Comment by jamesefhawkins** [comment]
We've since updated the slack welcome message
